mmedit/models/common/downsample.py

```diff
@@ -0,0 +1,21 @@
+def pixel_unshuffle(x, scale):
+    """Down-sample by pixel unshuffle.
+
+    Args:
+        x (Tensor): Input tensor.
+        scale (int): Scale factor.
+
+    Returns:
+        y (Tensor): Output tensor.
```

Comment:
The variable name is not necessary, see [here](https://github.com/open-mmlab/mmediting/blob/master/mmedit/models/common/flow_warp.py#L25).
```
Returns:
  Tensor: Output tensor.
```